tool/grind.dart

```diff
@@ -193,11 +191,11 @@ indexResources() {
     throw new StateError('lib/resources directory not found');
   }
   var outDir = new Directory(path.join('lib'));
-  var out = new File(path.join(outDir.path, 'src${sep}resources.g.dart'));
+  var out = new File(path.join(outDir.path, 'src', 'html', 'resources.g.dart'));
   out.createSync(recursive: true);
   var buffer = new StringBuffer()
-    ..write('// WARNING: This file is auto-generated.\n\n')
-    ..write('library resources;\n\n')
+    ..write('// WARNING: This file is auto-generated. Do not taunt.\n\n')
```

Comment:
This actually matches the contents of the file that's currently on disk – it wasn't my clever :smile: